src/apis/record.ts

```diff
@@ -9,8 +10,10 @@ import { baseInstance } from './instance'
 const MEMORY_RECORD_SIZE = 7
 const MEMORY_COMMENT_SIZE = 5
 
-export const getCategory = () => {
-  return baseInstance.get('/record/category')
+export const getCategory = async (categotyId: parentCategoryID) => {
```

Comment:
오타 있습니당!! 관련된 부분 전부 변경해주세요 :)
```suggestion
export const getCategory = async (categoryId: parentCategoryID) => {
```